docs/ManualDeploymentExtended.md

```diff
@@ -960,6 +1063,10 @@ Confirm wallet password:
 17. idle      18. catch   19. robot    20. clay
 21. resemble  22. angry   23. work     24. until
 ---------------END LND CIPHER SEED-----------------
+
+!!!YOU MUST WRITE DOWN THIS SEED TO BE ABLE TO RESTORE THE WALLET!!!
```

Comment:
Maybe it makes sense to highlight this even more using a ```:::warn``` block.

Reply:
That's actually just a copy of the console output from running the "lncli create" command. I guess another warn block in the doc could be added but probably redundant given they'll get the same double warning when they run the command?